src/block/utils/FortuneTrait.php

```diff
@@ -59,6 +60,8 @@ abstract protected function getFortuneDropsForLevel(int $level) : array;
 	 * @return Item[]
 	 */
 	protected function weightedDrops(Item $item, int $min, int $max, int $fortuneLevel) : array{
+		assert($min <= $max, "Minimum drop amount must be less than or equal to maximum drop amount");
```

Comment:
We generally don't use assert for argument validation. This should throw an InvalidArgumentException.